index.html

```diff
@@ -3,9 +3,20 @@
   <head>
     <title>EST UI Widget</title>
     <link rel="stylesheet" type="text/css" href="style.css">
-    <script type="text/javascript" src="javascript/est-widget.js"></script>
+
+      <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
I see what the problem is now. ok, let's ask users to include jQuery before including our widget code. It's not a big problem TBH.